src/main/java/bridge/model/BridgeGame.java

```diff
@@ -0,0 +1,47 @@
+package bridge.model;
+
+import bridge.utils.enums.MoveCommend;
+
+import static bridge.utils.enums.Const.*;
+
+
+/**
+ * 다리 건너기 게임을 관리하는 클래스
+ */
+public class BridgeGame {
+    private final Bridge bridge;
+    private Position currentPosition;
+
+    public BridgeGame(BridgeSize bridgeSize, BridgeMaker bridgeMaker){
+        this.bridge = new Bridge(bridgeMaker.makeBridge(bridgeSize.getValue()));
+        this.currentPosition = new Position(BRIDGE_INIT_NUMBER.getNumber());
+    }
+
+    /**
+     * 사용자가 칸을 이동할 때 사용하는 메서드
+     * <p>
+     * 이동을 위해 필요한 메서드의 반환 타입(return type), 인자(parameter)는 자유롭게 추가하거나 변경할 수 있다.
+     */
+    public MoveResult move(String moveCommend) {
+        MoveCommend direction = MoveCommend.fromString(moveCommend);
+        boolean success = false;
+        if (direction.isEqualTo(bridge.getDirection(currentPosition))) {
+            currentPosition.increment();
+            success = true;
+        }
+        return new MoveResult(success, direction);
+    }
```

Comment:
direction이 맞았는지를 getDirection 해서 비교하지 말고 currentPosition이랑 direction을 파라미터로 넘겨주고 bridge가 판단하도록 구현하면 어떨까요?
```suggestion
    public MoveResult move(String moveCommend) {
        MoveCommend direction = MoveCommend.fromString(moveCommend);
        boolean success = false;
        if (bridge.isRightDirection(currentPosition, direction)) {
            currentPosition.increment();
            success = true;
        }
        return new MoveResult(success, direction);
    }
```

Reply:
훨씬 간결하고 좋네요!